src/extension.ts

```diff
@@ -97,23 +97,19 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 	await clearRegisteredTutorials();
 
 	// register the default tutorial
-	const tutorialPath = path.join(context.extensionPath, './demos/markdown/didact-demo.didact.md');
-	const tutorialUri = vscode.Uri.parse(`file://${tutorialPath}`);
+	const tutorialUri = vscode.Uri.file(context.asAbsolutePath('./demos/markdown/didact-demo.didact.md'));
```

Comment:
this new API is nice. i was curious about it and discovered that this will work "only" in local VS Code (as it was before anyway).
When we will want to make it working in remote environment, we might need to use extensionUri and joinPath https://github.com/microsoft/vscode/issues/90208
mentioning it to have a trace of it. No needed action for this PR

Reply:
Spun off https://issues.redhat.com/browse/FUSETOOLS2-916 to investigate further as needed